modules/efk-logging-deploying-about.adoc

```diff
@@ -28,15 +23,19 @@ in the `openshift-logging` project.
 You can modify any of the following components upon install or after install
 
 Management state::
-Cluster logging is managed by the Cluster Logging Operator. You can place the Cluster Logging Operator into an _unmanaged_ state allowing an administrator to assume full control of individual
-component configurations and upgrades by changing the `managementState` from `Managed` to `Unmanaged`.
+Cluster logging is managed by the Cluster Logging Operator. The operator monitor the Cluster Logging CR and applies changes when they are made.   
+
+If you want to customize components of your cluster logging deployment, such as the Curator cronjob, the Elasticsearch CRD, the Kibana deployment, and the log collection daemon set, 
```

Comment:
The Curator cronjob, the Elasticsearch CRD, the Kibana deployment, and the log collection daemon set are managed by openshift-logging-operator. if you want to modify them directly rather than in clusterlogging/instance.
you must place the Cluster Logging Operator into an _Unmanaged_ state. Otherwise, the Cluster Logging operator overwrites the changes automatically. In an unmanaged deployment the Cluster Logging Operator does not overwrite these changes and does not respond to changes in the CR.

@mburke5678  How about use word above?

Reply:
The other Part LGTM.  I'd like merge it soon. It is painful to review document in github.

Reply:
@anpingli We are shifting all the 4.0 docs to the new 4.1 branch on Wednesday morning (EDT). We are not supposed to merge until after. But, I will ask if I can merge this before that move (and a few others of yours) to make reviewing easier.
cc @vikram-redhat